alpaca_trade_api/polygon/entity.py

```diff
@@ -93,14 +95,8 @@ def __init__(self, raw):
         ])
 
     def _raw_results(self):
-        results = self._raw.get('results')
-        if not results:
-            # this is not very pythonic but it's written like this because
-            # the raw response for empty aggs was None, and this:
-            # self._raw.get('results', []) returns None, not [] which breaks
-            # when we try to iterate it.
-            return []
-        return results
+        self._raw = self._raw or defaultdict(lambda: [])
```

Comment:
sorry for the late response (your mixed closed PR with pickle data brought this back to my attention)
didn't merge this back then because this is not answering the issue I wrote the initial code for.
the scenario in which there is a response but there is no `results` in the response. 

this code, in that case will result in an exception. an example: trying to do something like this: 
`api.polygon.historic_agg_v2('AAPL2', 1, 'day', _from='2019-01-01', to='2019-1-5').df`
note that is a request for a non existing symbol: `AAPL2`

this will be the result of this code:
![image](https://user-images.githubusercontent.com/7924802/103475343-1adc5a00-4db5-11eb-887d-1bf0f85afc25.png)